.github/workflows/workflow.yml

```diff
@@ -53,6 +54,8 @@ jobs:
     runs-on: [self-hosted, Linux, AWS]
     timeout-minutes: 10
 
+    if: ${{ !startsWith(inputs.config, '[') }}
```

Comment:
Will this show as a skipped job in the UI if it evaluates to false?

Reply:
Yeah, I added a note to the description just before your comment

> I would plan to eventually ship a v4 that requires the JSON and removes the configure job altogether

Reply:
https://github.com/Brightspace/Landlord/actions/runs/10491788428

Reply:
My beef with skipped jobs is that if there are any this UI defaults to not clicking "Hide all checks" when they've all completed, vs. when you have no skipped jobs it abbreviates itself which is really nice 
![image](https://github.com/user-attachments/assets/f1eeefce-f7c3-487f-b55b-1e70ee78b52a)

Reply:
So if the answer is yes -- what about making a breaking version bump and requiring the format change?

Reply:
(because I think skipping configure would be _amazing_)

Reply:
We could do both I guess? Like v3 it skips and v4 you have to take the format change?